semver/parse_comparator.ts

```diff
@@ -47,5 +47,5 @@ export function parseComparator(comparator: string): Comparator {
     }
     : ANY;
 
-  return { operator, semver };
+  return { operator, ...semver, semver };
```

Comment:
I forsee the output of this object being confusing to users during the deprecation period. But I don't see another viable way of doing this. Food for thought...